packages/opub-ui/package.json

```diff
@@ -55,6 +55,7 @@
     "@radix-ui/react-checkbox": "^1.0.1",
     "@radix-ui/react-collapsible": "^1.0.2",
     "@radix-ui/react-dialog": "^1.0.3",
+    "@radix-ui/react-icons": "^1.3.0",
```

Comment:
Don't use radix icons for now. rely on shopify polaris icons. 
remove this dependency